
////////////////////////////////Exceptions ///////////////////////////////////
void main(){

  List<String> names=['mahmoud'];
  int x=10;
  int y=0;
  num result=x/y;
    print(result);

try{
   print(names[3]);   //// exception  
}
catch(e){
  print(e);
}
 

}